silkworm/infra/grpc/client/dispatcher.hpp

```diff
@@ -27,12 +27,12 @@ namespace silkworm::detail {
 
 template <typename Executor>
 struct ExecutorDispatcher {
-    Executor executor_;
```

Comment:
this should be a private class member IMO

Reply:
done